src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -103,6 +107,19 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit, OnChanges
             this.profileService.getProfileInfo().subscribe((profileInfo) => {
                 this.localVCEnabled = profileInfo.activeProfiles?.includes(PROFILE_LOCALVC);
                 this.athenaEnabled = profileInfo.activeProfiles?.includes(PROFILE_ATHENA);
+
+                // The online IDE is only available with correct SpringProfile and if it's enabled for this exercise
+                if (profileInfo.activeProfiles?.includes(PROFILE_THEIA)) {
+                    this.theiaEnabled = true;
+
+                    // Set variables now, sanitize later on
+                    this.theiaPortalURL = profileInfo.theiaPortalURL ?? '';
+
+                    // Verify that Theia's portal URL is set
+                    if (this.theiaPortalURL === '') {
+                        this.theiaEnabled = false;
+                    }
```

Comment:
```suggestion
                    // Set variables now, sanitize later on
                    this.theiaPortalURL = profileInfo.theiaPortalURL ?? '';
                    // Verify that Theia's portal URL is set
                    this.theiaEnabled = this.theiaPortalURL !== '';
```

Reply:
There will be some more checks in the future. Thus, I'd like to keep the more verbose way of checking